app/src/main/java/com/example/unlibrary/util/SendNotification.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * SendNotification
+ *
+ * November 22, 2020
+ *
+ * Copyright (c) Team 24, Fall2020, CMPUT301, University of Alberta
+ */
+
+package com.example.unlibrary.util;
+
+import android.util.Log;
+import android.view.View;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.JsonObjectRequest;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SendNotification {
+
+    private static final String TAG = SendNotification.class.getSimpleName();
+    private final String FCM_API = "https://fcm.googleapis.com/fcm/send";
+    private final String serverKey = "key=" + "AAAAAkUm6fE:APA91bE_uyTQS3tH0vG_JG7qDTkwxqGvL9tmwlPo1mPhd8jwF714tTa_tJzq6Kg16MoqJotD3zAejCkvqN2xfjjaQ9qR_T4R6GxGctES6DNhlANWR7QtvDDMNfUzIys3OZK1SsNUzgSO";
+    private final String contentType = "application/json";
+
+    /**
+     * Create the JSONObject that will be sent to FCM
+     * @param  view, view from which the application context will be gotten.
```

Comment:
nit: run formatter, i think theres an extra space ?? 
```suggestion
     * @param view, view from which the application context will be gotten.
```

Reply:
yeah run formatter, theres some formatting discrepencies below hehe